src/FormElement/ItemSetSelect.php

```diff
@@ -44,4 +47,38 @@ public function applyToQuery(Query $query, array $data, array $formElementData):
             $query->addFacetFilter($formElementData['field_name'], array_filter($data['item_set_id']));
         }
     }
+
+    public function setApiManager(ApiManager $api): void
+    {
+        $this->api = $api;
+    }
+
+    public function getApiManager(): ApiManager
+    {
+        // var_dump(($this->api));
+        return $this->api;
+    }
+
+    public function stringifyData(array $data, array $formElementData) : string
```

Comment:
Search form elements should return an array too. We can't know how complex they might be and they might want to return several "query parts"